spec/rubocop/cop/style/if_unless_modifier_spec.rb

```diff
@@ -310,4 +310,17 @@
       expect(corrected).to eq 'puts "string", (1 if a)'
     end
   end
+
+  context 'if-end with conditional as body' do
+    let(:source) do
+      ['if condition',
```

Comment:
There should also be tests showing that a body with several expressions, one of which is `if` is not affected.